src/StoreKeeper/WooCommerce/B2C/Frontend/Handlers/OrderHookHandler.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace StoreKeeper\WooCommerce\B2C\Frontend\Handlers;
+
+use StoreKeeper\WooCommerce\B2C\Factories\LoggerFactory;
+use StoreKeeper\WooCommerce\B2C\Helpers\HtmlEscape;
+use StoreKeeper\WooCommerce\B2C\I18N;
+use StoreKeeper\WooCommerce\B2C\Imports\OrderImport;
+
+class OrderHookHandler
+{
+    public const STOREKEEPER_ORDER_TRACK_HOOK = 'storekeeper_order_tracking_message';
```

Comment:
can you add the information about this hook to `docs/Readme.md`

Reply:
Added to Readme.md